packages/gatsby-source-medium/README.md

```diff
@@ -0,0 +1,56 @@
+# gatsby-source-medium
+
+Source plugin for pulling data into Gatsby from unofficial Medium JSON endpoints.
```

Comment:
Oh hmmm… just looked at the JSON output and it only sends previews back? https://medium.com/webpack/latest?format=json

That changes things… this should be emphasized then in the README so as not to give the wrong impression.